src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpThurloeDAO.scala

```diff
@@ -57,7 +57,7 @@ class HttpThurloeDAO ( implicit val system: ActorSystem, implicit val executionC
         ))
     } map { response =>
       response.status match {
-        case StatusCodes.OK => ()
+        case StatusCodes.Created => ()
```

Comment:
thurloe swagger says it returns OK, but it actually returns Created in this case. for new registrations, this caused it to hit throwBadResponse and not mark registration complete or save the user in rawls.